examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py

```diff
@@ -0,0 +1,29 @@
+from dagster_airlift.core import combine_defs
+from dagster_airlift.dbt import dbt_defs
+from dagster_dbt import DbtProject
+
+from dbt_example.dagster_defs.lakehouse import (
+    defs_from_lakehouse,
+    lakehouse_existence_check,
+    specs_from_lakehouse,
+)
+from dbt_example.shared.load_iris import CSV_PATH, DB_PATH, IRIS_COLUMNS
+
+from .constants import dbt_manifest_path, dbt_project_path
+
+defs = combine_defs(
+    defs_from_lakehouse(
+        specs=specs_from_lakehouse(csv_path=CSV_PATH),
+        csv_path=CSV_PATH,
+        duckdb_path=DB_PATH,
+        columns=IRIS_COLUMNS,
+    ),
+    dbt_defs(
+        manifest=dbt_manifest_path(),
+        project=DbtProject(dbt_project_path()),
+    ),
+    lakehouse_existence_check(
+        csv_path=CSV_PATH,
+        duckdb_path=DB_PATH,
+    ),
+)
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +1 to +30]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->@benpankow this might give you more conviction on the API. It was super clean to delete all the `dag_defs` and `task_defs` calls from migrate and then reformat. <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23881?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->